package.json

```diff
@@ -17,7 +17,7 @@
     "dev": "turbo dev:compile dev:app",
     "dev:app": "turbo dev:compile dev:app",
     "dev:compile": "turbo dev:compile",
-    "dev:pack": "turbo dev:pack --concurrency 16",
+    "dev:pack": "rm -rfv ./packages/*/penumbra-zone-*.tgz && turbo dev:pack --concurrency 16",
```

Comment:
Nit: you don't need the `-r` option, because the shell globs are well specified, and will always result in a list of filenames.